src/adapter/base.cr

```diff
@@ -32,7 +37,8 @@ abstract class Granite::Adapter::Base
   end
 
   def log(query : String, params = [] of String) : Nil
-    Granite::ORM.settings.logger.info "#{query}: #{params}"
+    Granite::ORM.settings.logger.info query.gsub(SQL_KEYWORDS_REGEX, "\\0".colorize(:white).mode(:bold).to_s )
+    Granite::ORM.settings.logger.info params
```

Comment:
This line is just left dangling when it shows up in the logs. Can we put a label on it, or better format that somehow?

Reply:
We can label to something like `Granite::ORM.settings.logger.info "Params: #{params}"`